frontend/javascripts/oxalis/view/action-bar/starting_job_modals.tsx

```diff
@@ -503,8 +491,84 @@ function RunAiModelTab({ aIJobModalState }: { aIJobModalState: string }) {
   );
 }
 
+function AlignmentTab() {
+  const centerImageStyle = {
+    margin: "auto",
+    width: 150,
+  };
+  return (
+    <div>
+      <Space align="center">
+        <Radio.Button className="aIJobSelection" checked={true}>
+          <Card bordered={false}>
+            <Space direction="vertical" size="small">
+              <Row className="ai-job-title">Align Sections</Row>
+              <Row>
+                <img
+                  src={`/assets/images/${jobNameToImagePath.align_sections}`}
+                  alt={"Example of improved alignment of slices"}
+                  style={centerImageStyle}
+                />
+              </Row>
+            </Space>
+          </Card>
+        </Radio.Button>
+      </Space>
+      <AlignSectionsForm />
+    </div>
+  );
+}
+
+function ShouldUseTreesFormItem() {
+  const tracing = useSelector((state: OxalisState) => state.tracing);
+  const trees = tracing.skeleton ? Object.values(tracing.skeleton.trees) : [];
+  return (
+    <div>
+      <Form.Item
+        name="useAnnotation"
+        label={
+          <Space>
+            <div style={{}}>
+              Manual Matches{" "}
+              <Tooltip title="Please select whether the alignment should take connected skeleton nodes between adjacent sections as alignment guideline whenever available.">
+                <InfoCircleOutlined />
+              </Tooltip>
+            </div>
+          </Space>
+        }
+        valuePropName="checked"
+        rules={[
+          {
+            validator: (_rule, checked) => {
+              if (checked) {
+                if (tracing.annotationId === "") {
+                  return Promise.reject(
+                    "No annotation was found. Please create an annotation first.",
+                  );
+                }
+                if (
+                  tracing.skeleton == null ||
+                  trees.filter((tree) => tree.edges.edgeCount > 0).length === 0
+                ) {
+                  return Promise.reject(
+                    "No skeleton edges were found. Please create a skeleton with at least one edge first.",
+                  );
+                }
+              }
+              return Promise.resolve();
+            },
+          },
+        ]}
+      >
+        <Checkbox> Use manual matches from skeleton. </Checkbox>
```

Comment:
I think we decided to call every tree/skeleton text in the UI consistently tree, correct @philippotto? 

Sorry, for the confusion @Tobias314. Back in the early WK days we the trees were called skeletons and not trees. Nowadays we try to kepp the naming consistent and use tree at least in all UI related things. Due to code-wise historical reasons we cannot full switch from naming skeletons to trees. That's why one might encounter the name skeleton still in the code. Afaik it is still the dominant word for naming variable.


```suggestion
        <Checkbox> Use manual matches from trees. </Checkbox>
```